src/util.ts

```diff
@@ -180,4 +180,22 @@ export function arrayEquals1D(a: ArrayLike<any>, b: ArrayLike<any>) {
         }
     }
     return true;
+}
+
+/*
+ * Determines "C" order strides for a given shape array.
+ * Strides provide integer steps in each dimention to traverse an ndarray.
+ *
+ * NOTE: These strides here are distinct from numpy.ndarray.strides, which describe actual byte steps.
+ */
+export function getStrides(shape: number[]): number[] {
+    // adapted from https://github.com/scijs/ndarray/blob/master/ndarray.js#L326-L330
+    const ndim = shape.length;
+    const strides = Array(ndim);
+    let step = 1; // strides assumed to be contiguous, so init step is 1
```

Comment:
I would propose to move this comment to the function documentation. If we ever start playing with views this assumption may not be true anymore